common/metrics/defs.go

```diff
@@ -52,22 +53,22 @@ func (md metricDefinition) GetMetricUnit() MetricUnit {
 	return md.unit
 }
 
-func NewTimerDef(name string) metricDefinition {
-	return metricDefinition{name: name, unit: Milliseconds}
+func NewTimerDef(name string, opts ...Option) metricDefinition {
```

Comment:
maybe disable this lint? (in separate PR)

Reply:
Done